pkg/apis/kops/v1alpha2/instancegroup.go

```diff
@@ -162,6 +162,9 @@ type InstanceGroupSpec struct {
 	// InstanceInterruptionBehavior defines if a spot instance should be terminated, hibernated,
 	// or stopped after interruption
 	InstanceInterruptionBehavior *string `json:"instanceInterruptionBehavior,omitempty"`
+	// InstanceMetadata defines the configurations to configure instance metadata in an
+	// instance. This is currently used to require EC2 instance metadata v2 in AWS
```

Comment:
Should match the other API file.
```suggestion
	// InstanceMetadata defines the EC2 instance metadata service options (AWS Only)
```

Reply:
Will do this and squash the changes.

Reply:
Thanks. Please keep integration tests as a separate commit.

Reply:
yes. a little too late hahah but I ll rebase.

Reply:
Ah... sorry...